tools/skbstat.py

```diff
@@ -0,0 +1,996 @@
+#!/usr/bin/python
+# @lint-avoid-python-3-compatibility-imports
+#
+# skbstat Trace skb events(ingress/egress/consume/drop) of kernel.
+#         For Linux, uses BCC, eBPF. Embedded C.
+#
+# USAGE: skbstat [-h] [-i INTERVAL] [-s SADDR] [-d DADDR] [-N ADDR]
+#                  [-S SPORT] [-D DPORT] [-P PORT] [-p PROTO] [--input]
+#                                    [--output] [--consume]
+#
+# Copyright (c) 2021 Dongdong Wang <wangdongdong.6@bytedance.com>
+# Licensed under the Apache License, Version 2.0 (the "License")
+#
+# 10-MAR-2021  Dongdong Wang Created this.
+
+from __future__ import print_function
+from bcc import BPF
+import argparse
+import ipaddress
+from time import strftime
+from struct import unpack
+from time import sleep
+
+# arguments
+examples = """examples:
+    ./skbstat                               # dump all skb dropping backtraces
+    ./skbstat --ingress                     # dump skb ingress routines
+    ./skbstat --egress                      # dump skb egress routines
```

Comment:
egress -> output?

You may add an example command line like
    skbstat --consume
?

Reply:
ok, output is file.
I'll add some example of `skbstat --consume`.